atk/README.md

```diff
@@ -1,12 +1,41 @@
 # atk 
 [Project site](https://gtk-rs.org/) | [Online documentation](https://gtk-rs.org/docs-src/)
 
-__Rust__ bindings and wrappers for __Atk__.
+__Rust__ bindings and wrappers for __Atk__, part of [gtk-rs](https://github.com/gtk-rs/gtk-rs).
 
 ## Minimum supported Rust version
 
 Currently, the minimum supported Rust version is `1.48.0`.
 
+## Documentation
+
+ * [Rust API - Stable](https://gtk-rs.org/docs/atk/)
+ * [GTK Installation instructions](https://www.gtk.org/docs/installations/)
+
+## Using
```

Comment:
A table with the features and a mention of the minimum version of the C library that is supported would be useful too here :)

Reply:
That's just too annoying to maintain imo.

Reply:
Why? Not very different from the minimum supported Rust version

Reply:
It's very different: the msrv is one information and pretty easy to keep track of. The features are plenty.